DSL/com.larsreimann.safeds/src/main/kotlin/com/larsreimann/safeds/staticAnalysis/GoalCallToGoalCallablel.kt

```diff
@@ -0,0 +1,54 @@
+package com.larsreimann.safeds.staticAnalysis
```

Comment:
Is there a way to get rid of this duplication? Maybe make `SdsCall` a superclass of `SdsGoalCall`? Do all scoping rules and validations of `SdsCall` also apply to `SdsGoalCall`?

Reply:
we need to have a short meeting to discuss this, I am not sure how to do this or even if we should.